app/v2/templates/get_templates.py

```diff
@@ -0,0 +1,19 @@
+from flask import jsonify, request
+from jsonschema.exceptions import ValidationError
+
+from app import api_user
+from app.dao import templates_dao
+from app.schema_validation import validate
+from app.v2.templates import v2_templates_blueprint
+from app.v2.templates.templates_schemas import get_all_template_request
+
+
+@v2_templates_blueprint.route("/", methods=['GET'])
+def get_templates():
+    validate(request.args.to_dict(), get_all_template_request)
```

Comment:
I'm confused by this - it appears the request can have a template type filter - but this is not used in the query?